src/assets/styles/setup/_variables.scss

```diff
@@ -2,11 +2,14 @@
     // Colors
     //dark theme
     --surface: #1D2125;
+    --text:#9fadbc;
+    --border: #555;
     --board-header-background-color: #0000003d;
-    --dynamic-text: #FFFFFF;
+    --dynamic-text: #9fadbc;
```

Comment:
you change the color not good it will affect other places. add new variant to the new color